core/src/main/java/org/hibernate/ogm/util/configurationreader/impl/SimplePropertyReaderContext.java

```diff
@@ -96,6 +99,26 @@ else if ( configuredValue instanceof Number ) {
 		}
 	}
 
+	private Integer getAsInteger() {
```

Comment:
`getAsInt()` could be implemented on top of this:

    Integer value = getAsInteger();
    return value != null ? value : 0;